examples/nats-echo/main.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018-2019 The NATS Authors
+// Copyright 2018 The NATS Authors
```

Comment:
Actually, since you edit all these files, please add `-2020` if missing, on all the ones you edit.